app/src/main/java/org/fossasia/susi/ai/skills/skilldetails/SkillDetailsFragment.kt

```diff
@@ -454,6 +457,10 @@ class SkillDetailsFragment : Fragment(), ISkillDetailsView {
                 } else {
                     Toast.makeText(context, getString(R.string.toast_empty_feedback), Toast.LENGTH_SHORT).show()
                 }
+
+                updateFeedback()
+                setFeedback()
```

Comment:
Here is what i did, huh didnt touch the other lines up though dont know why git is tracing them as modified